plugins/phpmyadmin-rule-exclusions-before.conf

```diff
@@ -19,476 +19,943 @@
 # Generic rule to disable plugin
 SecRule TX:phpmyadmin-rule-exclusions-plugin_enabled "@eq 0" "id:9513099,phase:1,pass,nolog,ctl:ruleRemoveById=9513100-9513999"
 
+#
+# -=[ Cookies and setup (same for both URL formats) ]=-
+#
+
+# Session cookies whitelist + persistent cookies
+# These cookies may persist beyond sessions and may block a user when trying
+# to access phpMyAdmin for a second time.
+SecAction \
+    "id:9513100,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'phpmyadmin-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetById=941100;REQUEST_COOKIES,\
+    ctl:ruleRemoveTargetById=941120;REQUEST_COOKIES,\
+    ctl:ruleRemoveTargetById=942100;REQUEST_COOKIES:auto_saved_sql,\
+    ctl:ruleRemoveTargetById=942200;REQUEST_COOKIES:pmaAuth-1,\
+    ctl:ruleRemoveTargetById=942340;REQUEST_COOKIES:pmaAuth-1,\
```

Comment:
Does this assume there is only one server on the configuration?

Reply:
Probably. Can you show how cookies looks like for multiple server setup?

Reply:
`1` is the server ID, any unsigned integer should be allowed 
You can view cookies for a multi server setup on our demo server [`5.2.x-dev`](https://demo.phpmyadmin.net/QA_5_2/)

Reply:
Does this apply for both `pmaAuth-X` and `pmaUser-X`?

Reply:
Yes :)

Reply:
Fixed.